app/views/discussions/_discussion_context.html.haml

```diff
@@ -17,7 +17,7 @@
     - if can? :set_volume, @discussion
       - current_volume = @discussion_reader.volume
       .btn-group.dropdown.volume-dropdown
-        %button{type: 'button', class:"btn btn-left btn-default dropdown-toggle", 'data-toggle': "dropdown", 'aria-expanded': false}
+        %button{type: 'button', class:"btn btn-left btn-default dropdown-toggle", 'data-toggle' => "dropdown", 'aria-expanded' => false}
```

Comment:
I'm not sure what the cause of this is (a gem version difference maybe?), but I get compile errors when using this 'string': 'value' syntax. Rather just change it than have it trip me up anymore.